operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -59,6 +59,7 @@ kiali_defaults:
     image_version: ""
     ingress_enabled: true
     namespace: ""
+    #override_ingress_yaml:
```

Comment:
is this really an override or a customization?

Reply:
its really an override- you are completely overriding what the operator would have used.